editor-components/team/team-14/team14.tsx

```diff
@@ -183,19 +176,17 @@ class Team14 extends Team {
     const itemCount: number = this.getPropValue("itemCount");
 
     return (
-      <div className={this.decorateCSS("container")}>
-        <div className={this.decorateCSS("max-content")}>
+      <Base.Container className={this.decorateCSS("container")}>
+        <Base.MaxContent className={this.decorateCSS("max-content")}>
           {(title || subtitle) && (
-            <div className={this.decorateCSS("up-page")}>
-              {title && <h1 className={this.decorateCSS("title")}>{title}</h1>}
-              {subtitle && (
-                <p className={this.decorateCSS("subtitle")}>{subtitle}</p>
-              )}
-            </div>
+            <Base.VerticalContent className={this.decorateCSS("up-page")}>
+              {title && <Base.SectionTitle className={this.decorateCSS("title")}>{title}</Base.SectionTitle>}
+              {subtitle && <Base.SectionDescription className={this.decorateCSS("subtitle")}>{subtitle}</Base.SectionDescription>}
```

Comment:
subtitle will be a string is you use as_string true. so the inline editor will not appear on click